<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamWidget</class>
 <widget class="QWidget" name="ExamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ExamWidget{
	background-color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutMain" stretch="1,0">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layoutTable">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layoutLabelTable">
         <item>
          <widget class="QLabel" name="labelTable">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
           </property>
           <property name="text">
            <string>考生列表</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horSpacerLabelTable">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelTipDbClk">
           <property name="styleSheet">
            <string notr="true">color: gray;</string>
           </property>
           <property name="text">
            <string>双击成绩可显示详情</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutRight">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="boxExamInfo">
         <property name="title">
          <string>考试信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="layoutExamName" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelExamNameText">
              <property name="text">
               <string>试卷名称:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelExamName">
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutCurTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelCurTimeText">
              <property name="text">
               <string>当前时间:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelCurTime">
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutStartDateTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelStartDateTimeText">
              <property name="text">
               <string>开始时间:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelStartDateTime">
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutEndDateTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelEndDatetimeText">
              <property name="text">
               <string>结束时间:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelEndDateTime">
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutScoreInClient" stretch="1,0">
            <item>
             <widget class="QLabel" name="labelScoreInClientText">
              <property name="text">
               <string>考生端改分:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbbScoreInClient"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutState" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelStateText">
              <property name="text">
               <string>状态:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelState">
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutNetwork" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelNetworkText">
              <property name="text">
               <string>网络:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNetwork">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color: gray</string>
            </property>
            <property name="text">
             <string>若网络异常，可重开此页面重试</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetButtons" native="true">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnShowLog">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>显示日志</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verSpacerBtnsBottom">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetLog">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
